src/less/donut-chart/donut-chart.less

```diff
@@ -0,0 +1,185 @@
+.donut-chart {
+    align-items: center;
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    max-width: 305px;
+    position: relative;
+    width: 100%;
+}
+.donut-chart__container {
+    display: inline-block;
+    max-width: 120px;
+    width: 50%;
+}
+.donut-chart__stacked .donut-chart__container {
+    max-width: 190px;
+    width: 100%;
+}
+.donut-chart__container-vertical {
+    margin-bottom: 20px;
+    max-width: 200px;
+    width: 100%;
+}
+.donut-chart-legend__container {
+    display: inline-block;
+    width: 100%;
+}
+.donut-chart__stacked {
+    max-width: 190px;
+}
+.donut-chart__legend-left {
+    flex-flow: row-reverse nowrap;
+    justify-content: flex-start;
+    max-width: 450px;
+}
+.donut-chart__legend-left .donut-chart__container {
+    margin-left: 30px;
+    max-width: 190px;
+}
+.donut-chart__legend-right {
+    flex-flow: row nowrap;
+    justify-content: flex-start;
+    max-width: 450px;
+}
+.donut-chart__legend-right .donut-chart__container {
+    margin-right: 30px;
+    max-width: 190px;
+}
+.donut-chart__legend-right .donut-chart-legend__container {
+    max-width: 210px;
+    width: 50%;
+}
+.donut-chart__legend-left .donut-chart-legend__container {
+    max-width: 210px;
+    width: 50%;
+}
```

Comment:
Can we just do:

```
.donut-chart-legend__container {
   max-width: 210px;
   width: 50%
}
```

IMO, we should set all base variables and have the left/right ones just either `unset` variables or override them.